ci-operator/config/openshift/cluster-api/openshift-cluster-api-openshift-4.2.yaml

```diff
@@ -0,0 +1,60 @@
+tag_specification:
+  name: '4.2'
+  namespace: ocp
+promotion:
+  name: cluster-api
+  namespace: ocp-future
+base_images:
+  base:
+    name: '4.2'
+    namespace: ocp
+    tag: base
+binary_build_commands: make manager clusterctl
+canonical_go_repository: github.com/openshift/cluster-api
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: cluster-api
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  unit:
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: 100m
+      memory: 400Mi
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
```

Comment:
We need to switch to Go 1.12 RSN.

Reply:
Let's do it all at once for all components in a single PR